static/bidder-info/suntContent.yaml

```diff
@@ -1,13 +1,9 @@
-endpoint: "https://b.suntcontent.se/cds/rtb/bid?ssp=pbs"
+endpoint: "https://b.suntcontent.se/cds/rtb/bid?ssp={{.AccountID}}"
+aliasOf: "seedingAlliance"
 maintainer:
   email: tech@seeding-alliance.de
```

Comment:
Maintainer email is same as  `seedingAlliance`. Therefore no need to specify `seedingAlliance` in static/bidder-info/suntContent.yaml